galleries/users_explain/figure/backends.rst

```diff
@@ -11,17 +11,17 @@ Backends
 What is a backend?
 ------------------
 
-A lot of documentation on the website and in the mailing lists refers
-to the "backend" and many new users are confused by this term.
-Matplotlib targets many different use cases and output formats.  Some
-people use Matplotlib interactively from the Python shell and have
-plotting windows pop up when they type commands.  Some people run
-`Jupyter <https://jupyter.org>`_ notebooks and draw inline plots for
-quick data analysis. Others embed Matplotlib into graphical user
-interfaces like PyQt or PyGObject to build rich applications.  Some
-people use Matplotlib in batch scripts to generate postscript images
-from numerical simulations, and still others run web application
-servers to dynamically serve up graphs.
+Backends are used for displaying Matplotlib figures (see :ref:`figure-intro`),
+on the screen, or for writing to files.  A lot of documentation on the website
```

Comment:
```suggestion
on the screen, or for writing to files. A lot of documentation on the website
```

Reply:
Is this really a thing?  Old people need two spaces.

Reply:
But you removed the two spaces in the rest of the paragraph?